packages/paste-website/src/components/Input.tsx

```diff
@@ -2,18 +2,16 @@ import * as React from 'react';
 import styled from '@emotion/styled';
```

Comment:
I'm generally of the opinion that every component should eventually have some tests, even if it's just a snapshot.

Is it worth moving these to named files, given we'll create a `__test__` folder for each under it's own directory in components?

Reply:
Hmmm.. we can create sibling `Input.spec.ts` files in the same folder.  Thoughts?

Reply:
kinda messy, and isn't the convention `__test__` folders?

Reply:
It isn't too bad.  Kind of looks like https://github.com/modulz/radix/tree/master/packages/radix/src/components

Or we can still have a __tests__ folder with all the component filenames repeated like
```
| __tests__
   -| Input.tsx
   -| ...
| Input.tsx
```

Reply:
Reverted the changes in this PR for now.